io.github.androidvirtualpen.virtualpen.metainfo.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.androidvirtualpen.virtualpen</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+  <name>Android Virtual Pen</name>
+  <summary>Capture android pen input on Linux</summary>
+  <developer id="tld.vendor">
+    <name>Mathieu Beliveau</name>
+  </developer>
+  <description>
+    <p>Android Virtual Pen allows to send pen input data from your android device to a pc host where it emulates a virtual pen. What distinguishes this solution from others, is emphasis on low-latency to provide the best drawing experience.
+    The Android app (.apk) can be found at: https://github.com/androidvirtualpen/virtualpen/releases/download/0.1/virtual-pen.apk</p>
+  <p>Features</p>
+  <ul>
+    <li>Adjustable pen pressure threshold such that you can move the pen-cursor without triggering an unintended click/draw action</li>
+    <li>Adjustable pen pressure sensitivity such that you can customize the force needed to be applied</li>
+    <li>Eraser support</li>
+    <li>Switch between stretched and fixed display mode. In stretched mode, the android device screen is stretched and completely mapped to your PC primary screen. Note that this is required to fully reach the boundaries of your PC screen from your device but causes some slight distortion if the screen ratios (android vs PC) are not the same.
+In fixed mode, if the android device screen resolution fits into your PC screen, the android device display is centered and mapped on the host at a 1:1 ratio. This mode should incurr no distortion.</li>
+    <li>Detects available usb devices and automatically save your settings for each</li>
+    <li>Supports both X11 and Wayland</li>    
+  </ul>
+  </description>
+  <launchable type="desktop-id">io.github.androidvirtualpen.virtualpen.desktop</launchable>
+  <icon type="stock">io.github.androidvirtualpen.virtualpen</icon>
+  <branding>
+    <color type="primary" scheme_preference="light">#ff00ff</color>
+    <color type="primary" scheme_preference="dark">#993d3d</color>
+  </branding>
+  <content_rating type="oars-1.0" />
+  <categories>
+    <category>Office</category>
+    <category>Graphics</category>
+    <category>Education</category>    
+    <category>Presentation</category>
+  </categories>
+  <keywords>
+    <keyword>stylus</keyword>
+    <keyword>pen</keyword>
+    <keyword>wacom</keyword>
+    <keyword>Intuos</keyword>
+    <keyword>emulator</keyword>
+  </keywords>
+  <url type="homepage">https://github.com/androidvirtualpen/virtualpen</url>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/Mathieu-Beliveau/android-virtual-pen/main/android-virtual-pen-screenshot.png</image>
```

Comment:
Should be from a tag or a commit

Reply:
Ok, thanks for the precision. From reading the doc, I was a bit unsure. The `Acceptable, but should be submitted upstream` wording is a bit unclear. I went into the repo to see how other PRs were doing and quite a lot of them had these .desktop, icons files included. Maybe changing the wording in the doc to be really clear about this would help. In all cases, agreed, will change.

Reply:
the word is clear. if you can avoid it, don't do it. the files are in the repo upstream.

The acceptable is because there are a lot of packages that claim to support linux but don't and refuse to fix the lack of icon and desktop files. since the files here can be found upstream, the definitely shouldn't be here.

"I looked at other repo" : two wrongs don't make a right.

Reply:
"two wrongs don't make a right", agreed. But that was to emphasize that the doc might not be clear enough on this point if other people are also commonly doing this mistake.